src/languages/textToDisplay.json

```diff
@@ -106,9 +109,11 @@
      "micInstructions": "Asegurate de tener acceso al micrófono.",
      "uploadError":"Error al guardar",
      "uploadMessage":"Hubo un error guardando tu comentario.",
-     "internetError":"Por favor cheque su conexión de internet, si todo functiona correctamente es nuestra culpa, estamos trabajando en arreglarlo, disculpe"
-
-
+     "internetError":"Por favor cheque su conexión de internet, si todo functiona correctamente es nuestra culpa, estamos trabajando en arreglarlo, disculpe",
+     "undo": "Canclear",

```

Comment:
undo -> Cancelar, you wrote Canclear you were close...  fix that small typo and push please.

Reply:
oh wow... typo :/